Installing `@pulumi/std` npm package fails due to missing `scripts/install-pulumi-plugin.js` file

```
$ npm install @pulumi/std
npm ERR! code 1
npm ERR! path /Users/user/proj/node_modules/@pulumi/std
npm ERR! command failed
npm ERR! command sh -c -- node scripts/install-pulumi-plugin.js resource std 1.1.0
npm ERR! node:internal/modules/cjs/loader:1029
npm ERR!   throw err;
npm ERR!   ^
npm ERR!
npm ERR! Error: Cannot find module '/Users/user/proj/node_modules/@pulumi/std/scripts/install-pulumi-plugin.js'
npm ERR!     at Module._resolveFilename (node:internal/modules/cjs/loader:1026:15)
npm ERR!     at Module._load (node:internal/modules/cjs/loader:872:27)
npm ERR!     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:82:12)
npm ERR!     at node:internal/main/run_main_module:23:47 {
npm ERR!   code: 'MODULE_NOT_FOUND',
npm ERR!   requireStack: []
npm ERR! }
npm ERR!
npm ERR! Node.js v19.1.0

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/user/.npm/_logs/2023-06-02T11_52_25_940Z-debug-0.log
```

--- a/Makefile
+++ b/Makefile
@@ -42,7 +42,10 @@ build_nodejs_sdk: gen_nodejs_sdk
 		yarn run tsc --version && \
 		yarn run tsc
 	cp README.md LICENSE sdk/nodejs/package.json sdk/nodejs/yarn.lock sdk/nodejs/bin/
+	mkdir -p sdk/nodejs/bin/scripts/
+	cp sdk/nodejs/scripts/install-pulumi-plugin.js sdk/nodejs/bin/scripts/
 	sed -i.bak 's/$${VERSION}/$(VERSION)/g' sdk/nodejs/bin/package.json
+	rm sdk/nodejs/bin/package.json.bak
 
 build_python_sdk: gen_python_sdk
 	cd sdk/python/ && \

